src/components/smart/Table.vue

```diff
@@ -1,69 +1,322 @@
 <template>
-  <div class="overflow-auto border shadow-md rounded-md border-dividerDark">
-    <table class="w-full">
-      <thead>
-        <slot name="head">
+  <div class="flex flex-1 flex-col">
+    <div v-if="pagination" class="mb-3 flex items-center justify-end">
+      <HoppButtonSecondary
+        outline
+        filled
+        :icon="IconLeft"
+        :disabled="page === 1"
+        @click="changePage(PageDirection.Previous)"
+      />
+
+      <div class="flex h-full w-10 items-center justify-center">
+        <p>{{ page }}</p>
```

Comment:
The `p` element isn't necessary here.

Reply:
I think we can wrap it inside a `span` tag, it gives better accessibility since `page` is a dynamic text
wdyt @jamesgeorge007

Reply:
Sure, we can replace the `div` wrapper with a `span` that is more appropriate for inline content.